packages/login/test/login-form.test.js

```diff
@@ -216,7 +216,9 @@ describe('login form', () => {
     expect(login.disabled).to.be.true;
   });
 
-  it('should focus the username field', () => {
+  it('should focus the username field', async () => {
+    // Apply same delay that is used for auto-focus
+    await new Promise(requestAnimationFrame);
```

Comment:
Could also use `await nextFrame()` instead here and below.

Reply:
Thanks, updated in both test files.